CONTRIBUTING.md

```diff
@@ -44,14 +40,13 @@ To run *Benchmark.js* performance tests, press "Run Tests" button.
 1. Don't break too much. See [how many projects depend on this](https://www.npmjs.com/browse/depended/fast-json-patch).
 2. Make sure that the browser tests pass in Chrome, Firefox, Edge and IE
 3. Make sure that the NodeJS tests pass `npm install && npm run test`
-4. Call `grunt uglify`. This updates the minified files in the `dist` directory.
-5. Call `git status` to verify that the only pending changes in the repo are the result of the previous step
-5. Call `grunt bump:patch`, `grunt bump:minor` or `grunt bump:major`. This command:
- - increments the version number in the relevant files
- - commits changes to Git with the version number as the commit message
- - creates a Git tag wit the version
+4. Modify version number in `src/duplex.ts` and `src/core.ts`.
```

Comment:
To have it in source file for reference and debugging, but I believe we could get it automatically in build time.